scripts/cli/start_cli_testnet.sh

```diff
@@ -13,7 +13,8 @@ source "$HOME/.cargo/env"
 
 SCRIPT_PATH="$(dirname $0)"
 
-RUN_PARAMS="--host ac.testnet.libra.org --port 8000 "
+WAYPOINT="0:a17035961b97359bcc22b628c1e269f543b265220d22a17e4a5b4e3aed86855b"
```

Comment:
this is not enforced in any testing afaiu and is easy to break with new release

Reply:
Yea, the problem is that the value of this waypoint is determined by the parameters in the `prod.tfvars` (`num_validators`, `config_seed`), which resides in another (private) repo.

I realize now that at least for the testnet we need to decouple the waypoint publishing from the `testnet` branch in the libra repo because the prod configuration can change independently and we probably don't want to start cherry-picking the right waypoint to the `testnet` branch when it happens.

In some sense it's similar to the way we're notifying users about the changes in `consensus_peers.config.toml`, which is via community announcements on `community.libra.org`.
Let me remove it from here for now, thanks :)